sentences/pl/homeassistant_HassRespond.yaml

```diff
@@ -0,0 +1,31 @@
+language: pl
+intents:
+  HassRespond:
+    data:
+      - sentences:
+          - "(witaj|cześć)[ home assistant]"
```

Comment:
Maybe also add `hej`? It's used in wake words so should be quite common for people to try to use it?

Reply:
added